rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,180 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.3
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka (bukka@php.net), Larry Garfield (larry@garfieldtech.com)
+  * Status: Draft
+
+===== Introduction =====
+
+The Open Source project that develops the PHP language and its reference implementation uses an RFC process to discuss
+and vote on proposed changes. This was created mainly for user facing changes. It works well for that purpose but it is
+not that effective for deciding about purely technical changes that impact PHP internals and extension API's. It is not
+clear how it should resolve conflicts on the technical basis between contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src when there is a disagreements between core developers.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * contributor - Anyone who has already contributed to php-src.
```

Comment:
What is considered contribution? Is a submitted (but not yet accepted) patch considered a contribution? I think this needs to be tightened up a bit and made more explicit.

Reply:
I removed this definition as the parts that it was used were also changed so it is not used anywhere.